### PR TITLE
Feature/add sw

### PR DESCRIPTION
**Pull request by mattruddy**

### CODE REVIEW

**Comment by nrgapple** [comment]
This is sick

File: src/serviceWorker.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ type Config = {
 };
 
 export function register(config?: Config) {
-  if (process.env.NODE_ENV === 'production' && 'serviceWorker' in navigator) {
```
Comment (nrgapple) [inline src/serviceWorker.ts:29]: Nice

**Review by nrgapple — APPROVED** [review]

**Comment by nrgapple** [comment]
I don’t think the images are working still. When i add it to my home screen it still fonts show the icon

**Comment by mattruddy** [comment]
> I don’t think the images are working still. When i add it to my home screen it still fonts show the icon

works on desktop, maybe it just needs a diff size for mobile

**Pull request merged**